src/types.ts

```diff
@@ -261,3 +282,206 @@ export interface CreatedComment {
 export const CreatedCommentSpec: ObjectSpec<CreatedComment> = {
   id: int,
 };
+
+export interface CreateBugContent {
+  product: string;
+  component: string;
+  summary: string;
+  version: string;
+  description: string;
+  op_sys: string;
+  platform: string;
+  priority: string;
+  severity: string;
+  alias?: string[];
+  assigned_to?: string;
+  cc?: string[];
+  comment_is_private?: boolean;
+  comment_tags?: string[];
+  groups?: string[];
+  keywords?: string[];
+  qa_contact?: string;
+  status?: string;
+  resolution?: string;
+  target_milestone?: string;
+  flags?: SetFlag[];
+}
+
+export const CreateBugContentSpec: ObjectSpec<CreateBugContent> = {
+  product: string,
+  component: string,
+  summary: string,
+  version: string,
+  description: string,
+  op_sys: string,
+  platform: string,
+  priority: string,
+  severity: string,
+  alias: optional(array(string)),
+  assigned_to: optional(string),
+  cc: optional(array(string)),
+  comment_is_private: optional(boolean),
+  comment_tags: optional(array(string)),
+  groups: optional(array(string)),
+  keywords: optional(array(string)),
+  qa_contact: optional(string),
+  status: optional(string),
+  resolution: optional(string),
+  target_milestone: optional(string),
+  flags: optional(array(object(SetFlagSpec))),
+};
```

Comment:
Unless we want to validate the data locally before passing it to the API (which I don't think we do) this object is unneeded. Indeed it is not used elsewhere in this patch.

Reply:
Yeah, validating data before sending it to Bugzilla API isn't best thing to do. `CreateBugContentSpec` is removed, thanks.